desktop/windows/release-notes/index.md

```diff
@@ -19,12 +19,42 @@ This page contains information about the new features, improvements, known issue
 
 Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/projects/1){: target="_blank" rel="noopener" class="_"} to see what's coming next.
 
+## Docker Desktop 4.3.0
+2021-12-02
+
+> Download Docker Desktop
+>
+> [For Windows](https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){: .button .primary-btn }
+
+### Upgrades
+
+- [Docker Engine v20.10.11](https://docs.docker.com/engine/release-notes/#201011)
+- [containerd v1.4.12](https://github.com/containerd/containerd/releases/tag/v1.4.12)
+- [Buildx 0.7.1](https://github.com/docker/buildx/releases/tag/v0.7.1)
+- [Compose v2.2.1](https://github.com/docker/compose/releases/tag/v2.2.1)
+- [Go 1.17.3](https://golang.org/doc/go1.17)
+- [Kubernetes 1.22.4](https://github.com/kubernetes/kubernetes/releases/tag/v1.22.4)
+- [hub-tool 0.4.4](https://github.com/docker/hub-tool/releases/tag/v0.4.4)
+
+### Bug fixes and minor changes
+
+- Improved: Docker Desktop on Apple Silicon no longer requires Rosetta 2.
```

Comment:
I don't think we should call out the Apple silicon Rosetta 2 entry in the Windows release notes